src/main/kotlin/com/jetbrains/snakecharm/lang/psi/resolve/SnakemakeReferenceResolveProvider.kt

```diff
@@ -1,26 +1,24 @@
 package com.jetbrains.snakecharm.lang.psi.resolve
 
 import com.intellij.psi.util.QualifiedName
-import com.jetbrains.python.psi.PyPsiFacade
-import com.jetbrains.python.psi.PyQualifiedExpression
+import com.jetbrains.python.psi.*
 import com.jetbrains.python.psi.resolve.PyReferenceResolveProvider
 import com.jetbrains.python.psi.resolve.RatedResolveResult
 import com.jetbrains.python.psi.types.TypeEvalContext
 
 class SnakemakeReferenceResolveProvider : PyReferenceResolveProvider {
-    private val namesToFQN = hashMapOf("expand" to "expand") // TODO what is the qualified name of 'expand'?
+    private val namesToFQN = hashMapOf("expand" to "snakemake.io")
 
-    override fun resolveName(element: PyQualifiedExpression, context: TypeEvalContext): MutableList<RatedResolveResult> {
+    override fun resolveName(element: PyQualifiedExpression, context: TypeEvalContext): List<RatedResolveResult> {
         val facade = PyPsiFacade.getInstance(element.project)
-        val fullyQualifiedName = namesToFQN[element.name] ?: return mutableListOf()
-        val qName = QualifiedName.fromDottedString(fullyQualifiedName)
 
-        val resolveContext = facade.createResolveContextFromFoothold(element)
-        val resolvedModules = facade.resolveQualifiedName(qName, resolveContext)
+        val fullyQualifiedName = namesToFQN[element.name] ?: return emptyList()
```

Comment:
I assume that this resolve is provided for any *.py file even not related to snakemake files

Reply:
Done